### PR TITLE
Fix FreeBSD CI build

### PR DESCRIPTION
**Pull request by quantum5**

### CODE REVIEW

File: testsuite/helloworld/tests/cpp_badalloc/test.yml [hunk 1]
```diff
@@ -6,3 +6,4 @@ expect: [RTE, MLE]
 feedback:
 - std::bad_alloc
 - std::bad_array_new_length
+- ''
```
Comment (kiritofeng) [inline testsuite/helloworld/tests/cpp_badalloc/test.yml:9]: Can we separate this into `test.linux.yml` and `test.freebsd.yml`?

Comment (quantum5) [reply testsuite/helloworld/tests/cpp_badalloc/test.yml]: Done.

**Comment by quantum5** [comment]
Jenkins, retest this please.

**Comment by quantum5** [comment]
Jenkins, retest this please.

**Review by Xyene — APPROVED** [review]

**Pull request merged**